README.md

```diff
@@ -349,6 +349,10 @@ Specifies whether to use PCP version 1 or 2. Only accepts '1' or '2'. Defaults t
 Maximum amount of time that may elapse when trying to establish a connection to
 the broker in seconds. Defaults to 5 seconds.
 
+**max-message-size (optional)**
```

Comment:
This section of documentation is for the pxp-agent config specifically. I don't believe `max-message-size` is configurable there.

If you want to document it in the projects where it's configurable, seems like that should go into https://github.com/puppetlabs/pcp-broker/blob/master/doc/configuration.md, https://github.com/puppetlabs/clj-pcp-client/blob/master/README.md, and Orchestrator (I guess a PCP [ticket](https://tickets.puppet.com)).

Background on this are PCP-491 and PCP-480. The options were added in case we needed to be able to tweak them later, but the goal was to have messages be relatively small and if use-cases came up that violated that we'd try to solve them differently.